src/express-api/server.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const io = require('socket.io-client')
+const express = require('express')
+const cors = require('cors')
+
+const socket = io.connect('http://localhost:3000/caps')
+
+const PORT = process.env.PORT || 3001
+
+const app = express()
+app.use(cors())
+app.use(express.json())
+app.use(express.urlencoded({extended:true}))
+
+app.use(express.json())
+
+// Proof of life
+app.get('/', (req, res) => {
+  res.status(200).send('Hello World!')
+})
+
+// Pickup
+app.post('/pick-up', (req,res) => {
+  socket.emit('pickup', req.body)
+  res.status(200).json(req.body)
+})
+
+// In-transit
+app.post('/in-transit', (req,res) => {
+  socket.emit('in-transit', req.body)
+  res.status(200).json(req.body)
+})
+
+app.post('/delivered', (req,res) => {
+  socket.emit('delivered', req.body)
+  res.status(200).json(req.body)
+})
+
+// Start server
+module.exports = {
+  app: app,
+  start: () => app.listen(PORT, console.log(`Server started on Port ${PORT}`))
+}
```

Comment:
you can re-write this line as `app.listen(PORT, console.log(`Server started on Port ${PORT}`))`

Reply:
Implemented in https://github.com/MuckT/caps/pull/2/commits/6d571ee1458dcd806695cc3fa21dfce9e8cabe65.